### PR TITLE
[MediaElement] Removed line that pauses player before setting it to null.

### LINKED ISSUE #9525: [Bug] MediaElement Disposing exception when MainPage is changed on iOS
### Description

=================================================================
	Native Crash Reporting
=================================================================
Got a SIGSEGV while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================

=================================================================
	Native stacktrace:
=================================================================
	0x10dbcd815 - /Users/macdevdaden/Library/Developer/CoreSimulator/Devices/86CB2D85-7D2A-4118-9F55-58CD6F1C6CB3/data/Containers/Bundle/Application/D6BE2062-65DD-4159-B862-D02C3AACE324/ChatUIXForms.iOS.app/ChatUIXForms.iOS : mono_dump_native_crash_info
	0x10dbc1905 - /Users/macdevdaden/Library/Developer/CoreSimulator/Devices/86CB2D85-7D2A-4118-9F55-58CD6F1C6CB3/data/Containers/Bundle/Application/D6BE2062-65DD-4159-B862-D02C3AACE324/ChatUIXForms.iOS.app/ChatUIXForms.iOS : mono_handle_native_crash
	0x10dbd4aa8 - /Users/macdevdaden/Library/Developer/CoreSimulator/Devices/86CB2D85-7D2A-4118-9F55-58CD6F1C6CB3/data/Containers/Bundle/Application/D6BE2062-65DD-4159-B862-D02C3AACE324/ChatUIXForms.iOS.app/ChatUIXForms.iOS : mono_sigsegv_signal_handler_debug
	0x7fff52457b5d - /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libsystem_platform.dylib : _sigtramp
	0x1f02d01e02bc1e02 - Unknown
	0x7fff25723833 - /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/Foundation.framework/Foundation : NSKeyValueWillChangeWithPerThreadPendingNotifications
	0x7fff2093846d - /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/AVFoundation.framework/AVFoundation : -[AVPlayer setRate:withVolumeRampDuration:playImmediately:rateChangeReason:]
	0x7fff20938834 - /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/AVFoundation.framework/AVFoundation : -[AVPlayer setRate:]
	0x10de39319 - /Users/macdevdaden/Library/Developer/CoreSimulator/Devices/86CB2D85-7D2A-4118-9F55-58CD6F1C6CB3/data/Containers/Bundle/Application/D6BE2062-65DD-4159-B862-D02C3AACE324/ChatUIXForms.iOS.app/ChatUIXForms.iOS : xamarin_dyn_objc_msgSend
	0x114eefd3a - Unknown

=================================================================
	Basic Fault Address Reporting
=================================================================
Memory around native instruction pointer (0x7fff513fa890):0x7fff513fa880  08 5b 41 5e 41 5f 5d c3 48 85 ff 7e 09 48 8b 07  .[A^A_].H..~.H..
0x7fff513fa890  8a 40 1c 24 01 c3 31 c0 c3 48 85 ff 74 12 78 18  .@.$..1..H..t.x.
0x7fff513fa8a0  48 8b 07 48 85 c0 74 08 48 89 c7 e9 dc c7 00 00  H..H..t.H.......
0x7fff513fa8b0 
 48 8d 05 6c 09 02 00 c3 48 89 f8 48 c1 e8 3c 48  H..l....H..H..<H

=================================================================
	Managed Stacktrace:
=================================================================
	  at <unknown> <0xffffffff>
	  at ObjCRuntime.Messaging:void_objc_msgSend <0x000e9>
	  at AVFoundation.AVPlayer:Pause <0x00122>
	  at Xamarin.Forms.Platform.iOS.MediaElementRenderer:Dispose <0x00253>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform.iOS.VisualElementPackager:Dispose <0x002c5>
	  at Xamarin.Forms.Platform.iOS.VisualElementPackager:Dispose <0x00071>
	  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1:Dispose <0x0024a>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform.iOS.VisualElementPackager:Dispose <0x002c5>
	  at Xamarin.Forms.Platform.iOS.VisualElementPackager:Dispose <0x00071>
	  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1:Dispose <0x0024a>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform
.iOS.VisualElementPackager:Dispose <0x002c5>
	  at Xamarin.Forms.Platform.iOS.VisualElementPackager:Dispose <0x00071>
	  at Xamarin.Forms.Platform.iOS.VisualElementRenderer`1:Dispose <0x0024a>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform.iOS.ListViewRenderer:DisposeSubviews <0x00252>
	  at Xamarin.Forms.Platform.iOS.ListViewRenderer:DisposeSubviews <0x001a2>
	  at Xamarin.Forms.Platform.iOS.ListViewRenderer:DisposeSubviews <0x001a2>
	  at Xamarin.Forms.Platform.iOS.ListViewRenderer:DisposeSubviews <0x001a2>
	  at Xamarin.Forms.Platform.iOS.ListViewRenderer:Dispose <0x0021a>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform.iOS.DisposeHelpers:DisposeModalAndChildRenderers <0x0030f>
	  at Xamarin.Forms.Platform.iOS.Platform:CleanUpPages <0x0019a>
	  at Xamarin.Forms.Platform.iOS.PlatformRenderer:Dispose <0x0018a>
	  at Foundation.NSObject:Dispose <0x00074>
	  at Xamarin.Forms.Platform.iOS.Platform:Dispose <0x000fa>
	  at Xamarin.Forms.Platform.iOS.Platform:Syst
em.IDisposable.Dispose <0x00071>
	  at Xamarin.Forms.Platform.iOS.FormsApplicationDelegate:UpdateMainPage <0x001eb>
	  at Xamarin.Forms.Platform.iOS.FormsApplicationDelegate:ApplicationOnPropertyChanged <0x000fa>
	  at Xamarin.Forms.BindableObject:OnPropertyChanged <0x00118>
	  at Xamarin.Forms.Element:OnPropertyChanged <0x000b2>
	  at Xamarin.Forms.Application:set_MainPage <0x00417>
	  at ChatUIXForms.Helpers.Methods:Logout <0x001ea>
	  at <LogOut>d__7:MoveNext <0x006e2>
	  at System.Runtime.CompilerServices.AsyncVoidMethodBuilder:Start <0x001be>
	  at ChatUIXForms.Views.MainPage:LogOut <0x0037a>
	  at Xamarin.Forms.MenuItem:OnClicked <0x000ab>
	  at Xamarin.Forms.MenuItem:Xamarin.Forms.IMenuItemController.Activate <0x00183>
	  at SecondaryToolbarItem:<.ctor>b__1_0 <0x00084>
	  at UIKit.UIControlEventProxy:Activated <0x000b1>
	  at System.Object:runtime_invoke_void__this__ <0x000e5>
	  at <unknown> <0xffffffff>
	  at UIKit.UIApplication:UIApplicationMain <0x00259>
	  at UIKit.UIApplication:Main <0x000b2>
	  
at UIKit.UIApplication:Main <0x00132>
	  at ChatUIXForms.iOS.Application:Main <0x00092>
	  at <Module>:runtime_invoke_void_object <0x00352>
=================================================================
### Steps to Reproduce
Spam new page button in repro. 
(In my project's case, just one page change is enough to get the exception)

### Expected Behavior
New Page appears with video on it.
### Actual Behavior
Video isn't disposed properly.
### Basic Information

- Version with issue: 4.5.0-pre4
- Last known good version :N/A
- IDE:Visual Studio 16.4.4
- Platform Target Frameworks: 
  - iOS:  13.3
  - Android: N/A
  - UWP:  N/A
- Affected Devices: iPhone 8 simulator

### Reproduction Link

[VideoPlayer.zip](https://github.com/xamarin/Xamarin.Forms/files/4186895/VideoPlayer.zip)

### PR DESCRIPTION
**Pull request by StevenGranados**
### Description of Change ###
Removed line that pauses player before setting it to null.
I think the pause is there to stop background playback of the media element, but since the next line also seems to stop it, I removed the line.
I appreciate there is probably a lot more going on here though @peterfoot .
I'm not sure how to handle the video source, there has to be one for the exception to occur. It's currently taking a hard-coded URL source.

### Issues Resolved ### 
- fixes #9525 

### API Changes ###
 None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###

Will now not give an exception on Dispose.

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Have a page in iOS with a media element. Change page so that the media element is disposed.
### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by StevenGranados** [comment]
This is my first pull request, so I've almost certainly done something wrong on my UITest. Seeing as it is a simple fix on a new control, maybe best if the test is removed?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9525.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	[Category(UITestCategories.ManualReview)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 9525, "MediaElement Disposing exception when MainPage is changed on iOS", PlatformAffected.iOS)]
+	public class Issue9525 : TestNavigationPage
+	{
+		protected override void Init()
+		{
+#if APP
+			Device.SetFlags(new List<string>(Device.Flags ?? new List<string>()) { "MediaElement_Experimental" });
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9525.cs:23]: Yeah, I think we just need to move this stuff out of the #if APP.

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9525.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			return new ContentPage
+			{
+				Content = new StackLayout
+				{
+					Children =
+					{
+						new MediaElement
+						{
+							AutomationId = "Issue9525MediaElement",
+							Source="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4",
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue9525.cs:45]: We need to find a video that won't disappear on us. Something uploaded to the source, perhaps.

**Comment by StevenGranados** [comment]
https://github.com/xamarin/Xamarin.Forms/issues/9525#issuecomment-615854771
This comment seems to suggest a better fix than mine.

**Comment by rmarinho** [comment]
Hey @StevenGranados  can you fix the flag and upload a video so we can use on the repo? thanks

**Comment by StevenGranados** [comment]
I don't know how to put files in the source. I think there might have to be a new type of MediaSource to allow for it. At the moment there is Uri and File (I assume from the device file system).
I've tried to put a video in as an embedded resource, but I can't use streams in the Source property.
I've tried to put a video in each project separately (Resources folder and set to BundleResource) and refer to the filename in the Source property, but I'm not having any joy. @peterfoot any ideas?

**Comment by StevenGranados** [comment]
#11417 uses a Uri in the Source parameter. I assume you can control the availability of this?

**Comment by peterfoot** [comment]
@StevenGranados I think @rmarinho just meant a video online in a location which won't change. Personally I've always found this to be an issue, even when testing against Channel 9 videos e.g. https://sec.ch9.ms/ch9/80a3/6563611f-6a39-44fa-a768-1a58bdd080a3/HotRestart.mp4 So I'd suggest something like that which should hopefully be stable for months if not longer.

**Comment by StevenGranados** [comment]
@rmarinho I've made some changes

**Comment by PureWeen** [comment]
@peterfoot @StevenGranados  FYI

https://github.com/xamarin/Xamarin.Forms/pull/11862

**Pull request merged**